docs/filter-functions.md

```diff
@@ -33,19 +33,20 @@ The following functions are supported in addition to the built-in functions prov
 
 | Function | Input | Args | Output |
 | --------------- | ------- | ---- |  ---- |
-| [`allDocs`](#alldocs)<br />Checks the list includes only documents | [files](/context-variables.md#files) | - | Bool |
-| [`allImages`](#allimages)<br />Checks the list includes only images | [files](/context-variables.md#files) | - | Bool |
-| [`allTests`](#alltests)<br />Checks the list includes only tests | [files](/context-variables.md#files) | - | Bool |
-| [`codeExperts`](#codeexperts)<br />Get list of contributors based on expert reviewer model results| [`repo`](/context-variables.md#repo) | `gt`, `lt` | [String] |
-| [`estimatedReviewTime`](#estimatedreviewtime)<br />Estimated review time in minutes | [branch](/context-variables.md#branch-context)| - | Integer |
+| [`allDocs`](#alldocs)<br />Checks the list includes only documents | [`files`](./context-variables.md#files) | - | Bool |
+| [`allImages`](#allimages)<br />Checks the list includes only images | [`files`](./context-variables.md#files) | - | Bool |
+| [`allTests`](#alltests)<br />Checks the list includes only tests | [`files`](./context-variables.md#files) | - | Bool |
+| [`codeExperts`](#codeexperts)<br />Get list of contributors based on expert reviewer model results| [`repo`](./context-variables.md#repo) | `gt`, `lt` | [String] |
```

Comment:
are you sure? if it works in docs.gitstream.cm its better not to change it, as sometimes localhost behave different than production

Reply:
Yes, links to other pages don't work on https://docs.gitstream.cm/filter-functions
It returns 404 (hike here for example: https://docs.gitstream.cm/context-variables.md#files)